js/components/frame.js

```diff
@@ -433,6 +435,16 @@ class Frame extends ImmutableComponent {
     this.webview.addEventListener('media-paused', ({title}) => {
       windowActions.setAudioPlaybackActive(this.props.frame, false)
     })
+    this.webview.addEventListener('console-message', (e) => {
+      if (this.props.enableNoScript && e.level === 2 &&
+          e.message && e.message.includes(consoleStrings.SCRIPT_BLOCKED)) {
+        // Note that the site was blocked
+        // TODO: Parse out the location of the script that was blocked and send
+        // it too
+        windowActions.setBlockedBy(this.props.frame,
```

Comment:
Would prefer to store a list of URLs instead of the full message in case we need t use the info for something else later on.